scalafmt-core/shared/src/main/scala/org/scalafmt/config/AlignToken.scala

```diff
@@ -25,7 +25,9 @@ object AlignToken {
   implicit val DefaultAlignTokenDecoder: ConfDecoder[AlignToken] =
     ConfDecoder.instance[AlignToken] {
       case Conf.Str("caseArrow") => Ok(caseArrow)
-      case Conf.Str(regex) => Ok(AlignToken(regex, ".*"))
+      case Conf.Str(regex) =>
+        val owner = default.find(_.code == regex).fold(".*")(_.owner)
```

Comment:
looks good.

it probably doesn't play well with a `preset` until the metaconfig changes are merged, right?
let's say,
```
preset = defaultWithAlign
align.preset = ...
```
will mean we are not starting with `default`.

Reply:
`align.tokens.add` successfully overrides preset:

```
<<< ONLY respect specified owner
preset = defaultWithAlign
align.preset = most
align.tokens.add = [{code = "=", owner = "(Term.Assign|Defn.Val)"}]
===
object fmt {
  val header = JwtHeader(algorithm = Some(JwtAlgorithm.HMD5))
  val content = payload.toString
  val claim = JwtClaim(content = content)
}
>>>
object fmt {
  val header  = JwtHeader(algorithm = Some(JwtAlgorithm.HMD5))
  val content = payload.toString
  val claim   = JwtClaim(content    = content)
}
```